Cargo.toml

```diff
@@ -33,7 +33,7 @@ fdlimit = "0.1"
 ctrlc = { git = "https://github.com/paritytech/rust-ctrlc.git" }
 jsonrpc-core = { git = "https://github.com/paritytech/jsonrpc.git", branch = "parity-1.11" }
 ethcore = { path = "ethcore", features = ["work-notify", "price-info", "stratum"] }
-ethcore-bytes = { path = "util/bytes" }
+parity-bytes = { git = "https://github.com/paritytech/parity-common", rev = "1803c0f" }
```

Comment:
please remove `rev` from `Cargo.toml`. Version is already locked in `Cargo.lock` file. This field will only force us to use `sed` before every dependency update

Reply:
Yes, it's terrible. It's here because I could not for the world convince cargo to go out an fetch the latest version from git. When there are mismatching versions (or when rustc *thinks* the versions don't match) errors pop up like popcorn.

Do you happen to know how to force cargo to do a git fetch?

My plan here was to get the PR reviewed and then remove the `rev = …` as the last step before merging. In a follow-up PR I'll remove the crates now moved to `parity-common` and switch to using them from crates.io if I can't get git updates to work as expected.

Reply:
> Do you happen to know how to force cargo to do a git fetch?

For jsonrpc crates it's enough to do `cargo update -p X`, where X is the name of one of the crates from jsonrpc repo

Reply:
Ok, it's possible that I messed up (I did discover one instance of a crate getting fetched with `path` so perhaps that's what tripped me up yesterday).

I'll remove the `rev` so we can merge this and then we'll see what happens when ppl star using it.